src/js/structs/PodContainer.js

```diff
@@ -30,7 +30,7 @@ module.exports = class PodContainer extends Item {
 
       default:
         return Object.assign(Object.create(PodContainerStatus.NA), {
-          displayName: StringUtil.capitalize(this.get('status').toLowerCase())
+          displayName: StringUtil.capitalize(this.getStatusString().toLowerCase())
```

Comment:
Here we make it uppercase and then lowercase again. Maybe it would be better to omit `toUpperCase` in `getStatusString`